.github/workflows/build.yml

```
@@ -0,0 +1,63 @@
+name: build
+
+on: [push, pull_request]
+
+env:
+  FORCE_COLOR: 1
+  GH_TOKEN: ${{ secrets.SEMANTIC_RELEASE_GITHUB_TOKEN }}
```

Afaik GitHub automatically adds a secret "GITHUB_TOKEN" so it shouldn't be necessary to add one manually?
```suggestion
  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

---

That doesn't work with the version of `semantic-release` this repo is using, due the issues fixed in newer releases. @felixfbecker shall I bump node deps in this PR?

---

@felixfbecker ping

---

done